.github/pull_request_template.md

```
@@ -0,0 +1,342 @@
+</details>
+
+# Cross - QA
+
+<details>
+<summary>A continuación, se presentan las evidencias funcionales del desarrollo realizadas por un desarrollador diferente al que realizó el feature: </summary>
+</details>
+
+
+# Criterios de seguridad
+
+Certifico que mi solicitud satisface cada uno de los siguientes requerimientos de seguridad y no viola las buenas prácticas aqui definidas. Y de asi serlo, lo he declarado previamente con el equipo de desarrollo y los lideres  de toda el area mediante un mensaje escrito en slack o correo: 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)